docs/modules/ROOT/pages/how-tos/cloudscale/install.adoc

```diff
@@ -409,138 +395,111 @@ EOF
 terraform apply
 ----
 
-. Create the first shown DNS records
-
-. Wait for the DNS records to propagate!
-+
-[source,console]
-----
-sleep 600
-host "api.${CLUSTER_ID}.${BASE_DOMAIN}"
-----
+. Set up DNS NS records on parent zone using the data from the Terraform output variable `dns_entries` from the previous step
 
-. Provision master nodes
+. Create LB hieradata
 +
-[source,console]
+[source,bash]
 ----
 cat > override.tf <<EOF
 module "cluster" {
-  bootstrap_count = 1
+  bootstrap_count = 0
+  master_count    = 0
   infra_count     = 0
+  storage_count   = 0
   worker_count    = 0
 }
 EOF
-
-terraform apply
+terraform apply -target "module.cluster.local_file.lb_hieradata[0]"
 ----
 
-. Add the remaining shown DNS records to the previous ones.
+. Review and merge the LB hieradata MR (listed in Terraform output `hieradata_mr`) and wait until the deploy pipeline after the merge is completed.
 
-. Wait for bootstrap to complete
+. Create LBs
 +
-[source,console]
+[source,bash]
 ----
-openshift-install --dir ../../../target \
-  wait-for bootstrap-complete
+terraform apply
 ----
 
-. Remove bootstrap node and provision infra nodes
+
+////
+########################
+TODO fix command below
+########################
+////
+. Make LB FQDNs available for later steps
 +
-[source,console]
+.Store LB FQDNs in environment
+[source,bash]
 ----
-cat > override.tf <<EOF
-module "cluster" {
-  worker_count    = 0
-}
-EOF
-
-terraform apply
-
-export KUBECONFIG="$(pwd)/../../../target/auth/kubeconfig"
-
-# Once CSRs in state Pending show up, approve them
-# Needs to be run twice, two CSRs for each node need to be approved
-while sleep 3; do \
-  oc get csr -o go-template='{{range .items}}{{if not .status}}{{.metadata.name}}{{"\n"}}{{end}}{{end}}' | \
-  xargs oc adm certificate approve; \
+declare -a LB_FQDNS
+for id in 0 1; do
+  LB_FQDNS[$id]=$(terraform state show module.cluster.exoscale_domain_record.lb[$id] | grep hostname | cut -d'=' -f2 | tr -d ' "\r\n')
```

Comment:
```suggestion
  LB_FQDNS[$id]=$(terraform state show module.cluster.cloudscale_server.lb[$id] | grep name | cut -d'=' -f2 | tr -d ' "\r\n')
```

Reply:
almost, I needed to grep for fqdn, name would return more than one line